src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -958,6 +958,21 @@ class OptionsSegment(BaseSegment):
     )
 
 
+class ColumnDefinitionSegment(ansi.ColumnDefinitionSegment):
+    """A column definition, e.g. for CREATE TABLE or ALTER TABLE with support for passing
+    BigQuery column options"""
```

Comment:
I think the linter will complain about this. Needs to be <= 88 chars and end in full stop.

```suggestion
    """A column definition, e.g. for CREATE TABLE or ALTER TABLE.
    
    Override ANSI support for passing BigQuery column options
    """
```